### ISSUE TITLE
Validér postadresser med DAWA

### ISSUE THREAD

**Issue by Mortal** [TK-IT/web#65]
Der er et offentligt, cross-origin request-venligt API til at validere og udfylde danske postadresser: [Danmarks Adressers Web API - DAWA](https://dawa.aws.dk/).

Det har blandt andet mulighed for automatisk "adressevask", som finder den nærmeste beslægtede gyldige adresse ud fra en given adressetekst, sammen med en bud på om det er den rigtige adresse, den har fundet.

Det er også en mulighed at lave autocomplete i et administrationsinterface -- og det kan implementeres med AJAX-kald direkte til DAWAs servere uden at vores Django-server skal være inde over.

Man må højst lave [10 samtidige HTTP-requests](http://dawa.aws.dk/generelt#maxrequests) fra en given IP-adresse.

Implementationen af DAWA er desuden [open source på GitHub](https://github.com/DanmarksAdresser) og er tilsyneladende implementeret i Node.JS.

**Comment by furt11** [comment]
Nice! Men vi bør vel stadig have mulighed for at man kan have adresse uden for DK's grænser? Eller er vi kun interesseret i deres adresse, hvis de kan modtage post i DK?

**Comment by Mortal** [comment]
Nej, lad os blive ved med at gemme adresser i hele verden. De fleste bor dog i DK og jeg synes det giver mening at vaske vores danske adresser.
